diff --git a/node_modules/@parcel/runtime-browser-hmr/lib/loaders/hmr-runtime.js b/node_modules/@parcel/runtime-browser-hmr/lib/loaders/hmr-runtime.js
index c69e170..cc968a1 100644
--- a/node_modules/@parcel/runtime-browser-hmr/lib/loaders/hmr-runtime.js
+++ b/node_modules/@parcel/runtime-browser-hmr/lib/loaders/hmr-runtime.js
@@ -62,6 +62,9 @@ function Module(moduleName) {
     },
     dispose: function (fn) {
       this._disposeCallbacks.push(fn);
+    },
+    decline: function() {
+      // TODO
     }
   };
   module.bundle.hotData = undefined;
