diff --git a/node_modules/@parcel/runtime-browser-hmr/lib/loaders/hmr-runtime.js b/node_modules/@parcel/runtime-browser-hmr/lib/loaders/hmr-runtime.js
index 3c6d76a..6d71811 100644
--- a/node_modules/@parcel/runtime-browser-hmr/lib/loaders/hmr-runtime.js
+++ b/node_modules/@parcel/runtime-browser-hmr/lib/loaders/hmr-runtime.js
@@ -64,6 +64,9 @@ function Module(moduleName) {
     },
     dispose: function (fn) {
       this._disposeCallbacks.push(fn);
+    },
+    decline: function () {
+      // TODO
     }
   };
   module.bundle.hotData[moduleName] = undefined;
